src/applications/static-pages/facilities/FacilityListWidget.jsx

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+import { apiRequest } from '../../../platform/utilities/api';
+import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
+import { buildAddressArray } from '../../facility-locator/utils/facilityAddress';
+
+export default class FacilityListWidget extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      loading: true,
+      parsedFacilities: JSON.parse(this.props.facilities),
```

Comment:
I think I would parse this in the createFacilityListWidget file, before passing it into your React component. That creates a cleaner separation between the code that's dealing with the DOM and data stored in it, and your React code.

It'll also make it easier to unit test this component.